Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -391,6 +391,21 @@ Note that enabling Maglev will have a higher memory consumption on each Cilium-m
 to the default of ``loadBalancer.algorithm=random`` given ``random`` does not need the extra lookup
 tables. However, ``random`` won't have consistent backend selection.
 
+Maglev table size (M) per-service annotations
+=============================================
+
+To reduce the memory footprint of Maglev mode, it is possible to set the M
+value per service via a Kubernetes annotation. The annotation is
+``io.cilium/maglev-table-size``.
```

Comment:
@brb as a API to the user, I wonder whether we should rather have something like ``io.cilium.maglev/max-backends`` (naming tbd of course), and from that user input we derive internally the best suited Maglev table size? Or better to just go with raw table size, wdyt?